src/dmd/cparse.d

```diff
@@ -4300,13 +4303,23 @@ final class CParser(AST) : Parser!AST
      */
     private AST.Dsymbol applySpecifier(AST.Dsymbol s, ref Specifier specifier)
     {
+        //printf("applySpecifier() %s\n", s.toChars());
         if (specifier.alignExps)
         {
+            //printf("  applying _Alignas %s, packalign %d\n", (*specifier.alignExps)[0].toChars(), cast(int)specifier.packalign);
```

Comment:
This one and more others below can be taken out before merge.

Reply:
I generally leave debugging printf's in, rather than retype them every time I need them.